src/lib/provas/2019/32.json

```
@@ -0,0 +1,18 @@
+{
+    "ano": "2019",
+    "questao": "32",
+    "area": "Fundamentos",
+    "enunciado": "Sobre grafos, assinale a alternativa correta",
+    "img_url": [],
+    "alternativas": {
+        "A": "Um grafo ponderado é um grafo não direcionado em que todos os pares de vértices são adjacentes, \nisto é, há arestas ligando todos os vértices entre si. ", 
```

Here we go again "\n"

```suggestion
        "A": "Um grafo ponderado é um grafo não direcionado em que todos os pares de vértices são adjacentes, isto é, há arestas ligando todos os vértices entre si. ", 
```